ontobio/ontol.py

```diff
@@ -598,12 +598,49 @@ def logical_definitions(self, nid):
         else:
             return []
 
+    def definition(self, nid):
+        """
+        Text definition object for a node
+
+        Arguments
+        ---------
+        nid : str
+            Node identifier for entity to be queried
+
+        Return
+        ------
+        dict
+            definition object, dict(val=TEXT, xrefs=LIST)
+        """
+        return self._get_meta_prop(nid, 'definition')
+        
+    def definition_val(self, nid):
```

Comment:
the most common operation is getting the value; but in any case this will be dropped as I accidentlly reimplemented this code on a separate PR that's already been merged, doh